axol/axol.py

```diff
@@ -2093,6 +2095,31 @@ def love_mute(message):
         return True
 
 
+def catch(message):
+    if message.chat.id != neftepepe_chat:
+        return
+    if random.randint(0, 1000) != 228:
+        return
+    catching_message = bot.send_sticker(message.chat.id,
+                                        'CAACAgIAAxkBAAIfjWDhr5dRAZlK_8xkJUEvrepeAW7_AAL_DAACtTNJSgfaJegf7oIIIAQ')
+    timer = Timer(20, podcleanitb_sledy, [catching_message])
+    timer.start()
+    bot.register_next_step_handler(catching_message, gotcha, timer=timer)
+
+
+def gotcha(message, timer):
+    timer.cancel()
+    bot.send_sticker(message.chat.id, random.choice(config.catching_hands))
+
+
+def podcleanitb_sledy(message):
+    try:
+        bot.clear_step_handler(message)
```

Comment:
wow, wow

Reply:
a
eto drugoe

Reply:
dumal chto when next_step_handler otworkal, to pri try udalit him here bude exception
A you prosto timer otmenila

Reply:
Cho